test/boot.js

```diff
@@ -2,6 +2,10 @@
 
 import helperFixture from './lib/fixture';
 
+if (!window.MutationObserver) {
+  require('webcomponents.js/MutationObserver');
```

Comment:
Required to make the new prop type tests pass in IE. It will only error on the user if they try and use `slot()` without `MutationObserver`. This isn't a breaking change, but maybe we want to log if `MutationObserver` isn't available? Might be best to do this at some global level, or throw an error in the spot that requires it?